src/VariableStatementAnalysis.ts

```diff
@@ -1,3 +1,4 @@
 export interface VariableStatementAnalysis {
-    text: string;
```

Comment:
I started using this array to store variables with their values, you okay with that? or you want another array that stores something else beside the variables and their values?

Reply:
Text has been added back to variables
currently text doesn't get updated when resetting a variable, will fix that in the next push

Reply:
There should be a separate entry for each assignment.

i.e.
```typescript

let x = 1; // Line A
x = 2; // Line B

```

Should have two separate entries. since on line A `x` has the value of 1 and on line B x has the value of 2.